ari_backup/workflow.py

```diff
@@ -365,20 +365,28 @@ def run_command(self, command, host='localhost'):
         # wants to stop the workflow.
         raise
 
-      if stdout:
-        self.logger.debug(stdout)
-      if stderr:
-        # Warning level should be fine here since we'll also look at
-        # the exitcode.
-        self.logger.warning(stderr)
-
     if exitcode > 0:
       error_message = ('[{host}] A command terminated with errors and '
                        'likely requires intervention. '
                        'The command attempted was "{command}".').format(
-                           host=host, command=command)
+                           host=host, command=' '.join(command))
```

Comment:
Could you remove the double quotes around {command} and add a colon after "was"? I think that'll avoid ambiguity if {command} includes double quoted strings while still printing the command as a normal string rather than a list.